Flow.Launcher.Test/Plugins/ProgramTest.cs

```diff
@@ -8,23 +8,23 @@ namespace Flow.Launcher.Test.Plugins
     [TestFixture]
     public class ProgramTest
     {
-        [TestCase("Microsoft.WindowsCamera", "ms-resource:LensSDK/Resources/AppTitle", "ms-resource://Microsoft.WindowsCamera/LensSDK/Resources/AppTitle")]
```

Comment:
why is this entire class commented out?

Reply:
This test is for translating strings like `ms-resource://Microsoft.WindowsCamera/LensSDK/Resources/AppTitle` to the acutal title `Camera`, using interop. We retrieve titles from winrt api now so no longer needed.

Reply:
If not needed let's delete it?

Reply:
deleted